ivy/functional/backends/paddle/__init__.py

```diff
@@ -137,37 +137,37 @@
 
 # update these to add new dtypes
 invalid_dtypes = {
-    "2.4.2 and below": (
+    "2.4.0 and below": (
         ivy.uint16,
         ivy.uint32,
         ivy.uint64,
         ivy.bfloat16,
     ),
-    "2.5.1 and above": (
+    "2.4.1 and above": (
         ivy.uint16,
         ivy.uint32,
         ivy.uint64,
     ),
 }
 
 invalid_numeric_dtypes = {
-    "2.4.2 and below": (
+    "2.4.0 and below": (
         ivy.uint16,
         ivy.uint32,
         ivy.uint64,
         ivy.bfloat16,
     ),
-    "2.5.1 and above": (
+    "2.4.1 and above": (
         ivy.uint16,
         ivy.uint32,
         ivy.uint64,
     ),
 }
 
-invalid_int_dtypes = {"2.5.1 and below": (ivy.uint16, ivy.uint32, ivy.uint64)}
-invalid_float_dtypes = {"2.4.2 and below": (ivy.bfloat16,), "2.5.1 and above": ()}
-invalid_uint_dtypes = {"2.5.1 and below": (ivy.uint16, ivy.uint32, ivy.uint64)}
-invalid_complex_dtypes = {"2.5.1 and below": ()}
+invalid_int_dtypes = {"2.4.1 and below": (ivy.uint16, ivy.uint32, ivy.uint64)}
```

Comment:
These didn't need changes unless the behavior in version 2.5.1 has changed in which case it needs to be added to the keys here, so if this isn't the case the old keys were correct here.

Reply:
Sure. The behavior only changed for `invalid_float_dtypes` so I'll revert the changes for the others.